packages/comscore-analytics/src/index.ts

```diff
@@ -12,14 +12,16 @@ const comscoreAnalytics: ComscoreAnalytics = {
     ...analytics.actions,
     comscoreAnalytics: {
       pageview: ({ state }) => () => {
-        const { trackingIds } = state.comscoreAnalytics;
+        // Get Tracking ids from state.
+        const { trackingIds, trackingId } = state.comscoreAnalytics;
+        const ids = trackingIds || (trackingId && [trackingId]) || [];
 
-        if (trackingIds.length === 0)
+        if (ids.length === 0)
           warn(
-            "Trying to send pageviews to Comscore but `state.comscoreAnalytics.trackingIds` is empty."
+            "Trying to send an event to Comscore but neither `trackingId` nor `trackingIds` are specified inside `state.comscoreAnalytics`."
```

Comment:
Shouldn't this mention `pageview` instead of `event`?

Reply:
Woops, nice catch!